### DISCUSSION TITLE
Is Planetscale viable for single instance, multiple databases yet?

### DISCUSSION THREAD

**Discussion by j-fulbright** [planetscale/discussion#567] [Q&A] [answered]
I know there was some previous discussions about being able to run an instance with multiple databases, but hadn't seen any additional information. 

We currently are running on Aurora (MySQL) and are looking for some alternatives to our infrastructure, but due to the original design, we have some hefty needs. Being able to potentially have 1000s of DBs in one instance is where we'd need to exist, but could possibly shift away from that, but that is likely cost prohibitive on a service like PS, since each database would require a new per-month cost,

**Comment by mscoutermarsh** [comment, marked as answer]
Hi @j-fulbright, having multiple databases on a single PlanetScale DB is currently not possible.
